cli/tests/pcluster/config/test_section_cluster.py

```diff
@@ -555,6 +557,9 @@ def test_cluster_section_from_200_cfn(mocker, cfn_params_dict, expected_section_
     ],
 )
 def test_cluster_section_from_file(mocker, config_parser_dict, expected_dict_params, expected_message):
+    utils.get_default_values_for_required_clsuter_section_params(
+        config_parser_dict.get("cluster default"), only_if_not_present=True
```

Comment:
I think we can get rid of the `only_if_not_present` parameter by always adding a value in our file used for the tests.
It is set to true only here and I think it is more valuable to have the file populated with the required value than adding them here.

In the other cases `test_cluster_section_to_cfn`, `test_cluster_section_from_241_cfn`, `assert_param_validator` etc I agree this function is useful. 👍

Reply:
Okay. Makes sense.

Reply:
After looking again, it looks like it's also used in `assert_param_validator` so I'm inclined to keep it.